src/common/CommandLineUtils.ts

```diff
@@ -97,6 +97,14 @@ export class CommandLineUtils {
                         default: false
                     })
                 };
+            case FlagsConfigType.OutputFormatFlag:
+                return {
+                    outputFormat: Flags.option({
+                        char: 'f',
+                        description: messages.getMessage('outputFormatFlagDescription'),
+                        options: [OutputFormat.api] as const
```

Comment:
Usually we don't define flags of type Option but only provide 1 option.... Option-type flags are used when you have multiple choices to pick from.

I'm assuming that currently we will have two choices: either output as API or as normal/CLI 

So here we should have 2 choices `[OutputFormat.api, OutputFormat.cli]`. You can take it one step further and set the default flag value to  `OutputFormat.cli` 

If you set the default value it means that the caller can call a command with `-f` but without providing any value for the `-f` and it will pick the default value. If you don't set a default then whenever the caller calls a command with `-f` they must also provide a value for the `-f`

I think both approaches would be ok so pick one.